resources/src/main/java/org/robolectric/res/android/AttributeResolution10.java

```diff
@@ -346,8 +347,9 @@ public static void ApplyStyle(Theme theme, ResXMLParser xml_parser, int def_styl
                     value.set(entry.value);
                     source_style_resid = entry.style;
                     if (kDebugStyles) {
-                        ALOGI("-> From style: type=0x%x, data=0x%08x, style=0x%08x", value.get().dataType, value.get().data,
-                                entry.style);
+            ALOGI(
```

Comment:
Please fix these unwanted and incorrect code formatting.

Reply:
Sorry, I'll check again